templates/error.html

```diff
@@ -0,0 +1,85 @@
+{% extends "base.html" %}
+
+{% block meta_title %}
+  {% if error_code== 400 %}
```

Comment:
Instead of repeating this code here and below you can use the `set` to define a variable and use it throughout

Reply:
Good Suggestion! Thanks